handbook/marketing/index.md

```diff
@@ -27,4 +27,8 @@ See job descriptions and responsibilities of roles on the Marketing team:
 
 ## Definitions
 
-See "[Marketing definitions](https://docs.google.com/document/d/1h6xQeGHpMrsUUJ63Q6KupUNzq1iOpZfZSMC6ta3pLiY/edit#heading=h.pt4ztk7qhub0)".
\ No newline at end of file
+See "[Marketing definitions](https://docs.google.com/document/d/1h6xQeGHpMrsUUJ63Q6KupUNzq1iOpZfZSMC6ta3pLiY/edit#heading=h.pt4ztk7qhub0)".
+
+## [Marketing operations](marketing_operations.md)
```

Comment:
This page is becoming a bit odd... — a few random bullets on "messaging" at the top, then a bunch of headers with single lines beneath them saying "See X". 

Not sure what's best here, but consider moving this header up into the main Marketing bullets at the top?

Totally open to other opinions on this though... Clicking around, there's a mix of styles in our handbook.

Reply:
Yeah..I like the people ops design a lot better. Will make the change